### PR TITLE
fix-helm-chart-and-circle

### PR DESCRIPTION
**Pull request by calvix**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: helm/upgrade-schedule-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
```
Comment (njuettner) [inline helm/upgrade-schedule-operator/templates/deployment.yaml:1]:
```suggestion
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -6,9 +6,62 @@ orbs:
...
+          username_envar: "ALIYUN_USERNAME"
+          password_envar: "ALIYUN_PASSWORD"
+          requires:
+            - go-build
+          # Needed to trigger job also on git tag.
+          filters:
+            tags:
+              only: /^v.*/
+
+
```
Comment (njuettner) [inline .circleci/config.yml:54]:
```suggestion
```

**Review by njuettner — APPROVED** [review]

**Review by anvddriesch — APPROVED** [review]
LGTM

**Comment by calvix** [comment]
should be deployable now

**Pull request merged**